packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -1,7 +1,7 @@
 import { expect, haveResourceLike } from '@aws-cdk/assert';
+import { CertificateAuthority } from '@aws-cdk/aws-acmpca';
```

Comment:
Same here - no unqualified imports outside your module.

```suggestion
import * as acmpca from '@aws-cdk/aws-acmpca';
```